scripts/ZoneMinder/lib/ZoneMinder/Filter.pm

```diff
@@ -236,6 +236,11 @@ sub Sql {
                 || $term->{attr} eq 'Cause'
                 || $term->{attr} eq 'Notes'
                 ) {
+                if ( $term->{op} eq 'LIKE'
+                || $term->{op} eq 'NOT LIKE'
+                ) {
+                $temp_value = '%'.$temp_value.'%';
```

Comment:
This adds % around it regardless of if it contains %

Reply:
Yeah, very true. During testing SQL didn't seem to care about if % was already in the string, so adding extra logic to detect it didn't seem efficient. Do you think it best to detect if % is at the beginning and at the end and skipping this?

Reply:
To my mind, if I am specifying a % then I am interested in a specific case... like everything starting with something.. and if you add more %'s then I won't get that.  SO I think it is worth the extra processing which is pretty negligible.  
$temp_value = '%'.$temp_value.'%' if $temp_value !~ /%/;

Reply:
Cool, I'll make the change, let it run to ensure the filters are working and push the change.

Reply:
@connortechnology I've added the line and worked fine in my setup. Please review :)